### ISSUE TITLE
Can't restart plugin after end agent with Ctrl+C

### ISSUE THREAD

**Issue by WinMordekaiser** [bytedance/Elkeid#10]
**Describe the bug**
Can't restart plugin after end agent with Ctrl+C

**To Reproduce**
1. `cd /etc/hids/ && ./agent`
2. See the data output on the stdout of the current terminal
3. Execute `Ctrl+C` to stop agent 
4. `./agent`
5. I will get this message:
```
[{"data_type":"1001","level":"error","msg":"Plugin seems to be dead:\u0026{hids_driver 1.5.0.0 f30962ee9ee8fc025308fdf6098b2b48fec86c76274507317a47c89047e704a3 0xc00028c000 \u003cnil\u003e 0 4953 0 0 \u003cnil\u003e {[] 0}}","source":"config/config.go:76","timestamp":"1609404819"}]
```
6. If I stop agent and start it again, I will see the data output on the *stdout* of the current terminal

**OS information (please complete the following information):**
 - Distribution: centos
 - Version 7.8.2003
 - Kernel info 3.10.0-1127.10.1.el7.x86_64

**Comment by ZhangLei-cn** [comment]
It looks like some processes did not exit normally..
Execute the following command and try again:
```
killall hids_agent driver
rmmod hids_driver
```
Or you can print the log of plugin:
```
cat /etc/hids/plugin/driver/driver_r*.log
cat /etc/hids/plugin/driver/driver.stderr
```

**Comment by WinMordekaiser** [comment]
Here is the error log:
```
[2020-12-31 17:02:28.231761 +08:00] INFO [driver] src/main.rs:32: Crash check passed
[2020-12-31 17:02:28.232819 +08:00] INFO [driver] src/main.rs:40: Kernel version check passed
[2020-12-31 17:02:28.606701 +08:00] INFO [driver::prepare] src/prepare.rs:132: Last version is the same version
[2020-12-31 17:02:28.972471 +08:00] INFO [driver::prepare] src/prepare.rs:114: insmod hids_driver success
[2020-12-31 17:10:00.629023 +08:00] ERROR [driver] src/main.rs:88: IO error while reading marker: failed to fill whole buffer
```
But I can no longer reproduce this problem.
Thanks laoge.

**Comment by ZhangLei-cn** [comment]
Okay, I will try to reproduce this problem, it really does not seem right...
Plugin's log indicates that its connection was disconnected by the Agent. But Agent's log indicates that the plugin did not send a connection request.

**Comment by ZhangLei-cn** [comment]
Next version will fix it.

**Comment by ZhangLei-cn** [comment]
Has been fixed in 1.6.0.0.
